packages/mui-material/src/Accordion/Accordion.js

```diff
@@ -174,7 +191,7 @@ const Accordion = React.forwardRef(function Accordion(inProps, ref) {
       {...other}
     >
       <AccordionContext.Provider value={contextValue}>{summary}</AccordionContext.Provider>
-      <TransitionComponent in={expanded} timeout="auto" {...TransitionProps}>
+      <TransitionSlot in={expanded} timeout="auto" {...transitionProps}>
         <div
```

Comment:
I wouldn't like to add anything else to these PRs besides the deprecation. Otherwise, it might be hard to trace back the changes in the future. What do you think?

Reply:
Yeah agree for not handling it here, I would even go in the direction to splitting this PR, not add more things :D Let's keep the scope and add a note/issue about the proposed change.

Reply:
Alright, I'm fine to split it.